tests/api/v2/datadog/features/users.feature

```diff
@@ -0,0 +1,71 @@
+@v2 @endpoint(users)
+Feature: Users
+  Create, edit, and disable users.
+
+  Background:
+    Given a valid "apiKeyAuth" key in the system
+    And a valid "appKeyAuth" key in the system
+    And an instance of "Users" API
+
+  @todo
+  Scenario: Send invitation emails returns "OK" response
+    Given new "SendInvitations" request
+    And body {}
+    When the request is sent
+    Then the status is 201 OK
+
+  @todo
+  Scenario: Get a user invitation returns "OK" response
+    Given new "GetInvitation" request
+    When the request is sent
+    Then the status is 200 OK
+
+  Scenario: List all users returns "OK" response
+    Given new "ListUsers" request
+    When the request is sent
+    Then the status is 200 OK
+
+  Scenario: Create a user returns "OK" response
+    Given new "CreateUser" request
+    And body {"data": {"type": "users", "attributes": {"name": "Datadog API Client Python", "email": "{{ unique }}@datadoghq.com"}}}
+    When the request is sent
+    Then the status is 201 OK
+
+  Scenario: Disable a user returns "OK" response
+    Given there is a valid user in the system
```

Comment:
It may be helpful to quote "user" here so that its clearer there is a mapping between this valid user and the `user.data.id` from the `And` step